### ISSUE TITLE
[BUG][LOR] if the opponent fail to join the game, The LOR Match API returns the game as played and counts it a loss

### ISSUE THREAD

**Issue by dahmadjid** [RiotGames/developer-relations#409]
**Bug Description**  
I am actually not sure if this is a bug or it is intended so i apologize if it is not a bug
if the match doesn't start due to internet issue from player-1 or player-2, the match gets counted as actually played and returns both players decks,puuid,etc and the game would be counted as lost even though you wouldn't lose any LP as the match has not even started

**Problem Description**  
if the match that doesn't start gets counted, it would very hard to track data correctly, the winrates of the deck you queued up with becomes lower and thats an issue as it wont be accurate
It is also wasted resources as a request would be done for that specific match, on a larger scale that would be problematic 

**Expected Result**  

I expected the match that didn't start to not be counted as match that was played and not returned by LOR Match API

**Actual Result**  
a match that was not played was returned by the Match API

This issue can affect the accuracy of deck trackers (tracking winrates of deck) so anyone who would use your API in the future would suffer from this too

**Comment by paaksing** [comment]
Are you sure that the `game_outcome` is `"lose"` and not doing thing that evaluates to a boolean ?

**Comment by dahmadjid** [comment]
yes the `game_outcome` returns `loss` for 1 player and the other `win`

**Comment by RiotTuxedo** [comment]
Do you have an example match id?

**Comment by dahmadjid** [comment]
yes i do have 3 (idk why it happened to me this often but i didnt lose any lp nonetheless)
b5c7c0f6-d58c-45af-b2d3-0225e62c49e2
31d07120-1b58-4d38-8efe-42372edaef58
fe3975da-ce99-48f7-a1f7-a2746cbd848b
Europe region
